launch_ros/launch_ros/actions/load_composable_nodes.py

```diff
@@ -0,0 +1,202 @@
+        """
+        self.__composable_nodes_description = composable_nodes_description
+        self.__target_container = target_container
+        self.__in_parallel = None
+        if in_parallel is not None:
+            self.__in_parallel = normalize_to_list_of_substitutions(in_parallel)
+
+    def _load_node(
+        self,
+        composable_node_description: ComposableNode,
+        context: LaunchContext
+    ) -> None:
+        """
+        Load node synchronously.
+
+        :param composable_node_description: description of composable node to be loaded
+        :param context: current launch context
+        """
+        while not self.__rclpy_load_node_client.wait_for_service(timeout_sec=1.0):
+            if context.is_shutdown:
+                _logger.warn(
+                    "Abandoning wait for the '{}' service, due to shutdown.".format(
+                        self.__rclpy_load_node_client.srv_name
+                    )
+                )
+                return
+        request = composition_interfaces.srv.LoadNode.Request()
+        request.package_name = perform_substitutions(
+            context, composable_node_description.package
+        )
+        request.plugin_name = perform_substitutions(
+            context, composable_node_description.node_plugin
+        )
+        if composable_node_description.node_name is not None:
+            request.node_name = perform_substitutions(
+                context, composable_node_description.node_name
+            )
+        if composable_node_description.node_namespace is not None:
+            request.node_namespace = perform_substitutions(
+                context, composable_node_description.node_namespace
+            )
+        # request.log_level = perform_substitutions(context, node_description.log_level)
+        if composable_node_description.remappings is not None:
+            for from_, to in composable_node_description.remappings:
+                request.remap_rules.append('{}:={}'.format(
+                    perform_substitutions(context, list(from_)),
+                    perform_substitutions(context, list(to)),
+                ))
+        if composable_node_description.parameters is not None:
+            request.parameters = [
+                param.to_parameter_msg() for param in to_parameters_list(
+                    context, evaluate_parameters(
+                        context, composable_node_description.parameters
+                    )
+                )
+            ]
+        if composable_node_description.extra_arguments is not None:
+            request.extra_arguments = [
+                param.to_parameter_msg() for param in to_parameters_list(
+                    context, evaluate_parameters(
+                        context, composable_node_description.extra_arguments
+                    )
+                )
+            ]
+        response = self.__rclpy_load_node_client.call(request)
```

Comment:
It would be handy to emit an event with the id the container gave the node. It is needed to unload a node later, though I haven't figured out how to plumb that to a possible `UnloadComposableNode` action.

Reply:
Hmm, what's your specific use case? Something akin to ROS capabilities? I'd defer that to another PR, it sounds to me like a problem on its own.

Reply:
Specifically I'm looking for a way to test the `unload_node` service on the container process @mjcarroll is working on. +1 to deferring this and `UnloadComposableNode` to another PR.